out/index.js

```diff
@@ -0,0 +1,355 @@
+'use strict';
```

Comment:
this is automatically built from typescript, no need to review this file in-depth.